build/config/PublishData.json

```diff
@@ -23,7 +23,7 @@
         },
         "dev15.7.x-vs-deps": {
             "nugetKind": "PerBuildPreRelease",
-            "version": "2.7.*",
+            "version": "2.8.*",
```

Comment:
I thought we were considering going to 2.7.100 or something to get back in sync with VS?